src/backend/main.ts

```diff
@@ -1049,24 +1078,41 @@ ipcMain.handle(
       return false
     })
 
+    powerMonitor.removeListener('suspend', handleSuspend)
+    powerMonitor.removeListener('resume', handleResume)
+
     // Stop display sleep blocker
     if (powerDisplayId !== null) {
       logInfo('Stopping Display Power Saver Blocker', LogPrefix.Backend)
       powerSaveBlocker.stop(powerDisplayId)
     }
 
     // Update playtime and last played date
-    const finishedPlayingDate = new Date()
-    tsStore.set(`${appName}.lastPlayed`, finishedPlayingDate.toISOString())
-    // Playtime of this session in minutes
-    const sessionPlaytime =
-      (finishedPlayingDate.getTime() - startPlayingDate.getTime()) / 1000 / 60
+    currentSession.end = new Date()
+    playTimeSessions.push(currentSession)
+    tsStore.set(`${appName}.lastPlayed`, currentSession.end.toISOString())
+
+    const sessionPlaytime = playTimeSessions.reduce(
+      (acc, next) =>
+        acc + (next.end.getTime() - next.start.getTime()) / 1000 / 60,
+      0
+    )
```

Comment:
not a big deal, but I think it's better to do the division after you reduced the array

```suggestion
    const sessionPlaytime = playTimeSessions.reduce(
      (acc, next) =>
        acc + next.end.getTime() - next.start.getTime(),
      0
    ) / 60000
```